openedx/stanford/cms/djangoapps/contentstore/views/utility.py

```diff
@@ -9,14 +9,12 @@
 from edxmako.shortcuts import render_to_response
 from util.json_request import JsonResponse
 
-from contentstore.utils import reverse_course_url
+from openedx.stanford.cms.djangoapps.contentstore.views.helpers import reverse_course_url
 from xmodule.modulestore.django import modulestore
 from opaque_keys.edx.keys import CourseKey
 
 from student.auth import has_course_author_access
 
-__all__ = ['utility_handler']
```

Comment:
`__all__` is basically the list of things to pull in when you import `*`, like:

```py
from utility import *
# only pulls in `utility_handler`, not _everything_ in this file..
```

We only really needed to define `__all__` because we had cargo-culted the
wildcard import. Since we don't need that either, we can remove this now.